build.gradle

```diff
@@ -130,6 +130,8 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-common:${kotlin_version}"
     compile 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.7'
     compile "org.jetbrains:annotations:13.0"
+    compile "org.elasticsearch.client:elasticsearch-rest-client:${es_version}"
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
```

Comment:
What are these for

Reply:
kotlin-stdlib-jdk8 is for this line `client.threadPool().threadContext.stashContext().use {...}`

Reply:
removed rest-client dependency